.github/workflows/integration.yml

```diff
@@ -276,6 +274,7 @@ jobs:
       - name: Build Python PostgreSQL Driver
         shell: bash -l {0}
         env:
+          LD_PRELOAD: "$(gcc --print-file-name=libasan.so)"  # for sanitizer
```

Comment:
Probably want it on the test step below?

Reply:
Absolutely. yaml and I are NOT getting along today

Reply:
```
ERROR: ld.so: object '$(gcc' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
```

ok, should've expected that, but I guess instead of going in `env`, it has to go in `run` as `export LD_PRELOAD=...`. Sorry about the back-and-forth here.